src/style.rs

```diff
@@ -75,6 +75,11 @@ impl ProgressStyle {
         Ok(Self::new(Template::from_str(template)?))
     }
 
+    pub(crate) fn change_tab_width(&mut self, new_tab_width: usize) {
```

Comment:
I think it'd be easier if this was just named `set_tab_width()` as well, to match the public API.